dependencies.properties

```diff
@@ -23,9 +23,9 @@ version.gax_httpjson=0.92.1-SNAPSHOT
 # Versions for dependencies which actual artifacts differ between Bazel and Gradle.
 # Gradle build depends on prebuilt maven artifacts, while Bazel build depends on Bazel workspaces
 # with the sources.
-version.com_google_protobuf=3.18.1
+version.com_google_protobuf=3.19.1
```

Comment:
This causes the problem "The repository '@maven' could not be resolved and referenced by '@com_google_protobuf//java/util:util". gRPC is not the one causing it.

https://github.com/googleapis/gax-java/pull/1566

Reply:
We have to upgrade to the latest in protobuf too. The rest of the java client libraries will upgrade to protobuf 3.19.1 with shared deps 2.5.0.

Reply:
https://github.com/googleapis/gax-java/pull/1571 will fix the protobuf problem.

Reply:
1571 has been merged.